stage1/init/pod.go

```diff
@@ -168,10 +160,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool) error {
 	}
 
 	env := app.Environment
-	env.Set("AC_APP_NAME", name)
+	env.Set("AC_APP_NAME", appName.String())
```

Comment:
I see, ok, that's unfortunate - those should have been something more generic all along :-(. It is an ideal candidate to test the migration framework..